docs/modules/jenkins-shared-library/partials/odsComponentStageScanWithAqua.adoc

```diff
@@ -6,7 +6,7 @@ As a result, a Bitbucket Code Insight entry is added to the git commit (in Bitbu
 contains a link to the scan result on the Aqua platform. The Bitbucket Code Insight entry can be seen in a pull request.
 The pull request in Bitbucket shows the Code Insight of the latest commit of the PR.
 
-To get started, make sure you have a `ConfigMap` in your project-cd namespace in OpenShift that has these fields:
+To get started, make sure you have a `ConfigMap` in OpenDevStack project namespace (usually ods) in OpenShift that has these fields:
```

Comment:
I would add which should be the expected name of the cm (`aqua` AFAIK, right?)

Reply:
Yes, in the example is indicated that the name is aqua

Reply:
yes! saw it but just suggesting to make it explicit when using UI